misc/mc_event_trigger

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
> The key here is "optional": you need to add an option for the user to specify it

More triggers for other events, like PCIE, memory failure, etc, may be introduced, making the option too long. Is using the env config to enable/disable the trigger better?

Reply:
> > The key here is "optional": you need to add an option for the user to specify it
> 
> More triggers for other events, like PCIE, memory failure, etc, may be introduced, making the option too long. Is using the env config to enable/disable the trigger better?

Agreed. the best is to have trigger settings stored somewhere at /etc/sysconfig/rasdaemon, either at the env file or as a separate one.

My preference is that, instead of storing it as env vars, use a .ini format, e. g. something similar to:

```
# Always trigger script
[trigger /some/script/foo]
pcie = always

# Trigger only if the number of rate limit threshold of 100 events/second doesn't happen
[trigger /some/script/bar]
memory = 100

# trigger only once
[trigger /some/script/bar]
L1 = once
```

As this allow more flexibility of specifying thresholds to be used by the background thread/process that will be calling the script.

Reply:
Hi, mchehab:

I have pushed the new version to PR with two changes:
1. Move example script to contrib/ directory.
2. Store trigger settings at /etc/sysconfig/rasdaemon to control trigger.

Reply:
> typo: customuze->customise.
> 
> Also, Please the comment should be more generic, like:
> 
> > Allow users to optionally run a script when RAS events occur.
> 
> The key here is "optional": you need to add an option for the user to specify it, like:
> 
> ```
> rasdaemon --trigger-script /some/script
> ```
> 
> Also, instead of having a `misc/mc_event_trigger`, please provide an example script at `contrib/` directory.

It has fixed